spec/controllers/concerns/billable_event_trackable_spec.rb

```diff
@@ -44,20 +53,66 @@
   end
 
   describe '#track_billing_events' do
-    it 'does not fail if SpReturnLog row already exists' do
-      SpReturnLog.create(
+    it 'run track_billing_events and verify attributes' do
+      expect do
+        instance.track_billing_events
+      end.to change { SpReturnLog.count }.from(0).to(1)
+      sp_return_log = SpReturnLog.last
+      expect(sp_return_log.respond_to?(:profile_id)).to be true
+      expect(sp_return_log.respond_to?(:profile_verified_at)).to be true
+      expect(sp_return_log.respond_to?(:profile_requested_issuer)).to be true
```

Comment:
these assertions are basically checking if the migrations ran, I think we should check the values inside the columns to make sure they're what we expect.

For context, these values are what we're going to generate billing reports later on from, so we want to make sure they are correct

These are the assertions from later in the file
```suggestion
      expect(sp_return_log.profile_id).to eq active_profile.profile_id
      expect(sp_return_log.profile_verified_at).to eq active_profile.verified_at
      expect(sp_return_log.profile_requested_issuer).
        to eq active_profile.profile_requested_issuer
```